.travis.yml

```diff
@@ -4,24 +4,36 @@ cache: bundler
 language: ruby
 
 rvm:
-  - 2.5.8
-  - 2.6.4
-  - 2.7.1
+  - 2.5.9
+  - 2.6.9
+  - 2.7.5
```

Comment:
- citus uses travis, but kufu might have GitHub Actions or CircleCI as options.
- I'd also like to try Ruby 3.0 and 3.1

Reply:
> citus uses travis, but kufu might have GitHub Actions or CircleCI as options.

It's hard to keep up with a lot of diffs, do you switch CI?

> I'd also like to try Ruby 3.0 and 3.1

It's true, but I'd like to create other PR because the concerns are different.

Reply:
> It's hard to keep up with a lot of diffs, do you switch CI?

If you can enable travisci in kufu, no problem!

Reply:
We should move to GitHub Actions or CircleCI (I like GitHub Actions). But it is not point of this PR. 
Approve.